### ISSUE TITLE
bosh create release  givin Error 

### ISSUE THREAD

**Issue by gaurang90** [cloudfoundry/cf-release#1014]
Below is my cmd output of bosh create release command.

--------------------------------------------------------------

> Building dea_next...
>   No artifact found for dea_next
>   Generating...
>   Pre-packaging...
>   > + set -e -x
>   > + cd /tmp/d20160719-5670-1jcazpf/d20160719-5670-180td3f/dea_next
>   > + BUNDLE_WITHOUT=development:test
>   > + bundle package --all --no-install --path ./vendor/cache
>   > Fetching gem metadata from https://rubygems.org/.........
>   > Fetching version metadata from https://rubygems.org/..
>   > Fetching https://github.com/cloudfoundry/warden.git
>   > Fetching https://github.com/intridea/grape.git
>   > Fetching https://github.com/cloudfoundry/loggregator_emitter.git
>   > Fetching https://github.com/nats-io/ruby-nats.git
>   > Using i18n 0.7.0
>   > Using json 1.8.3
>   > Using minitest 5.8.4
>   > Using ffi 1.9.10
>   > Using httpclient 2.7.1
>   > Using json_pure 1.8.3
>   > Using bundler 1.11.2
>   > Using warden-protocol 0.1.5 from https://github.com/cloudfoundry/warden.git (at master@b72b8c5)
>   > Using loggregator_emitter 5.0.1 from https://github.com/cloudfoundry/loggregator_emitter.git (at master@7863578)
>   > Using warden-client 0.1.0 from https://github.com/cloudfoundry/warden.git (at master@b72b8c5)
>   > Using nats 0.6.0 from https://github.com/nats-io/ruby-nats.git (at master@d0556a0)
>   > Using em-warden-client 0.1.2 from https://github.com/cloudfoundry/warden.git (at master@b72b8c5)
>   > Using grape 0.16.3 from https://github.com/intridea/grape.git (at master@c2778cf)
>   > Updating files in vendor/cache
>   >   * rake-11.1.2.gem
>   > Could not find i18n-0.7.0.gem for installation
> 'dea_next' pre-packaging failed.

Please help me to solve this.

Thanks

**Comment by Amit-PivotalLabs** [comment]
ping @fraenkel @swetharepakula @ScarletTanager

**Comment by DanLavine** [comment]
Hi @gaurang90,

It looks like you might have something messed up with your ruby environment on your machine. Our CI pipelines do a fresh deploy of cf every time and are building successfully. 

Looking at your output I see "Fetching https://github.com/nats-io/ruby-nats.git Using i18n 0.7.0" and then you get an error for the same exact gem.

I would try running `gem install rake -v 11.1.2` and see if there are any issues when you try to install the gem manually. If that works then maybe you have a bad bundler version. There was an issue with with an early 1.12.x version (cannot remember which one exactly), but version 1.12.5 works fine.

Hope this helps,
@DanLavine, RuntimeOG Team

**Comment by gaurang90** [comment]
its working now after i reinstall ruby and able to successfully deploy cloud foundry. but i cant ping or curl at "api.bosh-lite.com". can anyone help me with this??

**Comment by DanLavine** [comment]
Hi @gaurang90,

This issue is probably coming from the fact that `bosh-lite.com` resolves to a local IP address on your machine which is a 10.* address. To fix this issue try running the `bin/add-route` script in the bosh-lite repository you downloaded from github. 

Seeing as how your original issue was fixed I'm going to close this issue. If the bosh-lite fix does not work, you could open up a new issue or ask a general question in the Cloud Foundry slack channel.

Best,
@DanLavine, RuntimeOG Team

**Comment by ivandavidov** [comment]
Although this issue has been long closed, I'd like to report that I experienced exactly the same problem with the most recent CF release. I had to manually install all reported missing gems in order to continue with the process. All gems and their corresponding versions for ``dea_next`` are located here:

```
cf-release/src/dea_next/Gemfile.lock
```

That behavior could be due to some misconfiguration from my end and since I managed to resolve the problem, I'm not reopening this issue.
